interpreter/cling/include/cling/Interpreter/Value.h

```diff
@@ -23,9 +21,66 @@ namespace llvm {
 namespace clang {
   class ASTContext;
   class QualType;
-  class RecordDecl;
 }
 
+// FIXME: Merge with clang::BuiltinType::getName
+#define BUILTIN_TYPES                                                  \
```

Comment:
As this is supposed to be "exported" from `Value.h`, could you please prefix this, say with `CLING_` or similar? Without it's probably too generic.

Reply:
That's done.